department-of-veterans-affairs/next-build
# Events layout followup tasks

## Description
Relates to #[)](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16399). 

Changes:
- Fixes reduntant typescript types
- Adds events modifications from [this](https://github.com/department-of-veterans-affairs/content-build/pull/1831/files) PR
- CTA string format helper function added
- Home breadcrumb modifications
- Updated logic around recurring events in Events layout
- Puts mostRecentDate data calculation in a react hook so dates are calculated in browser

## Testing done
Local testing with local cms data changes

## Screenshots
<img width="991" alt="Screen Shot 2023-12-27 at 10 35 45 AM" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/4ad46077-3d4d-4d2d-93af-ade8737d27d4">
<img width="816" alt="Screen Shot 2023-12-27 at 10 36 02 AM" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/b2747281-e18e-49ee-b84f-8fcc7bfc1d89">
<img width="816" alt="Screen Shot 2023-12-27 at 11 22 09 AM" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/d86f4a9f-5855-4df0-b92d-82dbf856ad57">
<img width="816" alt="Screen Shot 2023-12-28 at 10 56 54 AM" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/3142c2d1-bc76-4a69-a36f-0934dfa811c9">
<img width="816" alt="Screen Shot 2023-12-28 at 10 57 12 AM" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/99cbc696-8ef0-4773-bf29-9dcf1a3a8d22">


## QA steps
- View events page with recurring events (ie: /central-iowa-health-care/events/47347/)
   - Verify page renders correctly
   - Verify "view other times" button works as expected
   - Verify "show all times" button works as expected
   - Verify breadcrumbs match prod breadcrumbs


## Is this PR blocked by another PR?
- Add the `DO NOT MERGE` label
- Add links to additional PRs here:
